Adds Optimizely Documentation (#9968)
bbc/simorgh: src/app/hooks/useClickTrackerHandler/README.md

3. Log separate click events for each of a number of elements on click.

```jsx
/*
 * Example 1 - Log 1 click event when a component is clicked.

>  * In this example, one click event is triggered when the a tag is is clicked within the component and it will bring the user to the linked page.

Reviewer: ```suggestion
 * In this example, one click event is triggered when the anchor tag is clicked within the component and it will bring the user to the linked page.
```

  )
};

/*
 * Example 2 - Log 1 click event when any of the buttons are clicked

>  * In this example, if any of the buttons are clicked a click event will

Reviewer: ```suggestion
 * In this example, if any of the buttons are clicked, a click event will
```

    </ol>
  );
};

/*
 * Example 4 - Log 1 click event for both ATI and Optimizely when a component is clicked.

>  * In this example, one click event is triggered when the a tag is is clicked within the component and it will bring the user to the linked page.

Reviewer: ```suggestion
 * In this example, one click event is triggered when the anchor tag is clicked within the component and it will redirect the user to the url.
```

 */

import { useContext } from 'react';

> import  withOptimizelyProvider  from  '#app/containers/PageHandlers/withOptimizelyProvider';

Reviewer: Double spacing here.

 */

import { useContext } from 'react';

> import  withOptimizelyProvider from '#app/containers/PageHandlers/withOptimizelyProvider';

Reviewer: Double spacing between `import withOptimizelyProvider` too 😄
